### DISCUSSION TITLE
EMQX 5 single node: disable ports 4370 and 5370

### DISCUSSION THREAD

**Discussion by michaelcplusplus** [emqx/emqx#10806] [Q&A]
Hello,

I am running the EMQX Broker as a single node on a Windows server.
I want to run the broker as secure as possible
Is there any way to disable port 4370 and 5370.

In case the two ports can not be disabled:
Can the ports be bound to the local network interface so that they cannot be reached from outside.

Best regards
Michael

**Comment by lafirest** [comment]
@michaelcplusplus 
It seems that you can't disable these two ports, all of them are used for the communication between the EMQX cluster.
The `4370` is for the `ekka` and the `5370` is for the `gen_rpc`, you can see [here](https://www.emqx.io/docs/en/v5.0/deploy/cluster/intro.html) for details.

**Reply by michaelcplusplus** [reply]
OK. But I don't use the EMQX cluster mode.  The broker is operated as a single node
